packages/core/src/sequence.ts

```diff
@@ -16,14 +16,51 @@ import {
 import {parseOperationArgs} from './parser';
 import {writeResultToResponse} from './writer';
 import {HttpError} from 'http-errors';
-
+/**
+ * The default implementation of Sequence.
+ *
+ * This class implements default Sequence for the LoopBack framework.
+ * Default sequence is used if user hasn't defined their own Sequence
+ * for their application.
+ *
+ * Sequence constructor() and run() methods are invoked from [[http-handler]]
+ * when the API request comes in. User defines APIs in their Application
+ * Controller class.
+ *
+ * Sequence executes these steps
+ *  - Finds the appropriate controller method, swagger spec
+ *    and args for invocation
+ *  - Parses HTTP request to get API argument list
+ *  - Invokes the API which is defined in the Application Controller
+ *  - Writes the result from API into the HTTP response
+ *
+ * User can bind their own Sequence to app as shown below
+ * ```ts
+ * app.bind('sequence').toClass(MySequence);
+ * ```
+ */
 export class Sequence {
+   /**
+   * Constructor: Injects findRoute, invokeMethod & logError
+   * methods as promises.
+   *
+   * @param findRoute Finds the appropriate controller method,
+   *  spec and args for invocation
+   * @param invoke Invokes the method
+   * @param logError Logs error
+   */
   constructor(
     @inject('findRoute') protected findRoute: FindRoute,
     @inject('invokeMethod') protected invoke: InvokeMethod,
     @inject('logError') protected logError: LogError,
   ) {}
-
+  /**
+   * Runs the sequence
```

Comment:
@ritch @crandmck sendError(..) and sendResponse() are not exported methods .. hence they are not APIs. They are local wrapper functions which @bajtos implemented in this class. I will remove API docs for these methods since they are not APIs. If end-user implements their own sequence, they should be using writeResultToResponse(..) and logError(..) instead..